auth/auth.go

```diff
@@ -196,22 +196,35 @@ func (r *RenewingChecker) GetDedicatedIPServices() ([]DedicatedIPService, error)
 	return dipServices, nil
 }
 
-// fetchSaveServices fetches services and updates data appropriately
-func (r *RenewingChecker) fetchSaveServices(userID int64, data *config.TokenData) error {
-	services, err := r.creds.Services()
-	if err != nil {
-		return err
-	}
-
+// This will return the VPN service expiration date in the format YYYY-MM-DD HH:MM:SS
```

Comment:
[nit] might be also worth to mention when an error is returned, something like:
`FindVpnServiceExpiration returns VPN service expiration date in the format of YYY-MM-DD HH:MM:SS if there is matching VPN service ID, an error otherwise`